pkg/domain/attributor.go

```diff
@@ -99,46 +98,50 @@ func (n *CloudAssetDetails) UnmarshalJSON(data []byte) error {
 // returns either a 404 Not Found response, or a 200 OK response with no results
 type AssetNotFoundError struct {
 	Inner          error
-	AssetID        string
-	ScanTimestamp  string
 	AssetInventory string
+	Code           int
+	ScanTimestamp  string
 }
 
 func (err AssetNotFoundError) Error() string {
-	return fmt.Sprintf(
-		"Result not found for asset with ID %s as of scan time %s in asset inventory %s: %v",
-		err.AssetID, err.ScanTimestamp, err.AssetInventory, err.Inner)
+	return "Result not found for asset using asset inventory"
```

Comment:
Suggestion: how does the string "result not found in asset inventory" sound instead? Also, for best sorting and grouping results, we should probably follow some rules like "all lower case, no punctuation" for these types of strings.